### PR TITLE
Sales report

### PR DESCRIPTION
**Pull request by juliagolemo**
Issue: #16

### CODE REVIEW

File: public/images/kreciol.png
Comment (SzymonCogiel) [inline public/images/kreciol.png:1]: Are you sure that after moving tab icon from `public/kreciol.png` to `public/images/kreciol.png` is still working?

Comment (juliagolemo) [reply public/images/kreciol.png:1]: Everything is working

File: sales33.png
Comment (SzymonCogiel) [inline sales33.png:1]: Move this file

Comment (juliagolemo) [reply sales33.png:1]: done

**Review by SzymonCogiel — APPROVED** [review]

**Pull request merged**